pages/serverless-containers/faq.mdx

```diff
@@ -173,6 +173,20 @@ Integration is straightforward. Serverless Functions and Containers can be trigg
 
 You can copy an image from an external registry using the Docker CLI, or open source third-party tools such as [Skopeo](https://github.com/containers/skopeo). Refer to the [dedicated documentation](/serverless-containers/api-cli/migrate-external-image-to-scaleway-registry/) for more information.
 
+### How do Serverless Containers namespaces and Container Registry namespaces interact?
+
+Serverless Containers namespaces and Container Registry namespaces observe the following behaviors:
+
+- Creating a Serverless Containers namespace implicitly creates an empty Container Registry namespace.
+
+- Creating a Container Registry namespace **does not** create a Serverless Containers namespace.
+
+- Serverless Containers and Container Registry namespace are not linked.
+
+- With Serverless Containers, you can use images from any Container Registry namespace, as long as they are in the same [Project](/organizations-and-projects/concepts/#project).
+
+- If you delete The Container Registry namespace associated with a Serverless Containers namespace, it will be created again when deploying a container within this Serverless Containers namespace.
```

Comment:
```suggestion
- If you delete the Container Registry namespace associated with a Serverless Containers namespace, it will be created again when deploying a container within this Serverless Containers namespace.
```